crates/nu-parser/README.md

```diff
@@ -93,6 +93,6 @@ When the command runs, it will now be able to evaluate the whole math expression
 
 As some Shapes can consume multiple tokens, it's important that the parser allow for multiple Shapes to coexist as peacefully as possible.
 
-The simplest way it does this is to ensure there is at least one token for each required parameter. If the Signature of the command says that it takes a MathExpression and a Number as two required arguments, then the parser will stop the math parser one token short. This allows the second Shape to consume the final token.
+The simplest way it does this, is to ensure there is at least one token for each required parameter. If the Signature of the command says that it takes a MathExpression and a Number as two required arguments, then the parser will stop the math parser one token short. This allows the second Shape to consume the final token.
```

Comment:
I feel like for this sentence in particular the previous one flowed better.

Reply:
Looks like you are right. It didn't flow for me when I read it ;-)